Conformance results for v1.25/wke

wwentland opened the pull request:
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

For a full list of requirements, please refer to these sections of the docs: [_Contents of the PR_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr), and [_Requirements_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#requirements).

taylorwaggoner commented:
@wwentland thanks for submitting! Before we can merge, please include more details about the installation and cluster creation process in your README files. See our FAQ for more details: https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions

Also, please update the link to the logo to a version that is not white - we find that white logos tend not to show up very well. 

Thanks!

wwentland commented:
Thank you for your comments, @taylorwaggoner! I'll get them addressed as soon as I'm back from holiday.

wwentland commented:
@taylorwaggoner I've addressed the issues you raised and would appreciate a re-review of this submission.

In particular:

- Included additional information on cluster creation in the README
- Replaced the link with one that links to a non-white logo

Thanks!

taylorwaggoner commented:
You are now Certified Kubernetes
